### ISSUE TITLE
import/no-unresolved - False Positive

### ISSUE THREAD

**Issue by cusxio** [AtomLinter/linter-eslint#602]
![image](https://cloud.githubusercontent.com/assets/6487613/15955021/3a226c9c-2f10-11e6-900f-97d4a083be47.png)

It seems that there are problems when `linter-eslint` is trying to resolve modules. I'm not too sure how to debug this. When I run eslint in the terminal, the modules are able to resolve correctly.

```bash
$ DEBUG=eslint:cli-engine ./node_modules/.bin/eslint .
```

**Output**
```bash
/Users/cusx/Documents/dev/test/hello.js
  4:1  warning  Unexpected console statement  no-console
  5:1  warning  Unexpected console statement  no-console

✖ 2 problems (0 errors, 2 warnings)
```

Running on Atom 1.8.0 with Linter-Eslint 7.2.2

**Folder Structure**
```bash
├── .eslintrc.yml
├── goodbye.js
├── hello.js
├── node_modules
└── package.json
```

**.eslintrc.yml**
```yaml
extends: airbnb
installedESLint: true
plugins:
  - react
```

**goodbye.js**
```js
export default 'goodbye';
```

**hello.js**
```js
import React from 'react'; // unable to resolve path to module 'react'
import goodbye from './goodbye';  // unable to resolve path to module './goodbye'

console.log(goodbye);
console.log(React);
```

**package.json**
```json
  "devDependencies": {
    "eslint": "^2.11.1",
    "eslint-config-airbnb": "^9.0.1",
    "eslint-plugin-import": "^1.9.0",
    "eslint-plugin-jsx-a11y": "^1.3.0",
    "eslint-plugin-react": "^5.1.1"
  },
  "dependencies": {
    "react": "^15.1.0"
  }
```

**Comment by IanVS** [comment]
Hm, I haven't had this problem, though I also use `eslint-plugin-import`.

Could you share the `linter-eslint` value of your `~/.atom/config.cson`?

**Comment by cusxio** [comment]
@IanVS, here you go.

A weird thing that is happening is that older projects on my system, i.e. projects that are a few months old that are also using `eslint-plugin-import` seems to work fine.

**This issue only occur on newly created projects on my system.**

```cson
  "linter-eslint":
    disableFSCache: true
    disableWhenNoEslintConfig: false
    globalNodePath: "/Users/cusx/.nvm/versions/node/v5.10.1"
```

**Comment by IanVS** [comment]
Hmmm, have you restarted Atom recently?  Sometimes that seems to fix random problems.

**Comment by cusxio** [comment]
@IanVS interestingly restarting my computer fixes some `eslint-plugin-import` issue.

Seems like this is a problem on per-project basis.

**Comment by IanVS** [comment]
@cusxio do you think this issue can be closed, then?  Or do you still believe it to be a problem with `linter-eslint`?

**Comment by cusxio** [comment]
@IanVS 

I am not too sure at this point. I'll close this issue for now until I have something more concrete!
